src/main/app/src/assets/i18n/nl.json

```diff
@@ -111,7 +111,7 @@
   "bestandstype.onbekend": "Onbekend bestand",
   "bouwjaar": "Bouwjaar",
   "achternaam": "Achternaam",
-  "voorvoegsel": "Voorvoegsel",
+  "voorvoegsel": "Tussenvoegsel",
```

Comment:
Sounds quite confusing. If this is really a tussenvoegsel shouldn't we rename the field itself too?

Reply:
Ok! From a Domain Driven Design perspective this does go a bit against the 'ubiquitous language' concept but oh well.. That is pretty much impossible to achieve anyway in this domain..